pyaci/core.py

```diff
@@ -61,18 +61,20 @@ def _elementToString(e):
 
 aciMetaDir = os.path.expanduser(os.environ.get('ACI_META_DIR', '~/.aci-meta'))
 
+# The the folder does not exist: initialize the aciClassMetas as an empty dict
```

Comment:
I think that this could just be simplified like so.

```python
aciMetaFile = os.path.join(aciMetaDir, 'aci-meta.json')
if os.path.exists(aciMetaFile):
    with open(aciMetaFile, 'rb') as f:
        logger.debug('Loading meta information from %s', aciMetaFile)
        aciMeta = json.load(f)
        aciClassMetas = aciMeta['classes']
else:
    aciClassMetas = dict()
```

What do you think?

Reply:
Yes let me update to that then, simpler and it works !